libraries/Microsoft.Bot.Builder.AI.QnA/Schemas/Microsoft.QnAMakerDialog.uischema

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "https://schemas.botframework.com/schemas/ui/v1.0/ui.schema",
+  "flow": {
+    "widget": "ActionCard",
+    "body": "=action.hostname"
```

Comment:
I don't think this got added to the uischema definition in your other PR.

Reply:
Yes, that's a problem I need to confirm.

In ui.schema, `"additionalProperties"` is declared to `true` to enable additional fields as widget props. Based on the design of flow uischema, all other additional properties except `widget` will be passed to widget component as its props, so this is a loose restriction on ui.schema side.

As an example
```
<Widget ...additionalProps />
```

Reply:
[Portal](https://github.com/microsoft/BotFramework-Composer/blob/main/Composer/packages/extension-client/src/types/flowSchema.ts) to the ts version definition